src/Microsoft.Health.Fhir.Tests.Common/TestFiles/Normative/CodeSystem-careplan-category.json

```diff
@@ -3,27 +3,42 @@
   "id": "careplan-category",
   "text": {
     "status": "generated",
-    "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\"><h2>US Core CarePlan Category Extension Codes</h2><div><p>Set of codes that are needed for implementation of the US-Core profiles. These codes are used as extensions to the FHIR and US Core value sets.</p>\n</div><p>This code system http://hl7.org/fhir/us/core/CodeSystem/careplan-category defines the following codes:</p><table class=\"codes\"><tr><td style=\"white-space:nowrap\"><b>Code</b></td><td><b>Display</b></td><td><b>Definition</b></td></tr><tr><td style=\"white-space:nowrap\">assess-plan<a name=\"careplan-category-assess-plan\"> </a></td><td>Assessment and Plan of Treatment</td><td>The clinical conclusions and assumptions that guide the patient's treatment and the clinical activities formulated for a patient.</td></tr></table></div>"
+    "div": "<div>PlaceHolder</div>"
   },
   "url": "http://hl7.org/fhir/us/core/CodeSystem/careplan-category",
-  "version": "3.0.0",
+  "version": "4.1.0",
```

Comment:
We do have R4 specific folder for test files.  But if this file works well for all versions of FHIR (including STU3), then changing it here in Normative is fine.